images/macos/helpers/Xcode.Helpers.psm1

```diff
@@ -121,4 +121,29 @@ function Get-XcodePairsList {
         $result += "$watchName $phoneName"
     }
     return $result
+}
+
+function Invoke-XCVersion {
+    param(
+        [Parameter(Mandatory)]
+        [string] $Command,
+        [Parameter()]
+        [int] $RetryAttempts = 7,
+        [Parameter()]
+        [int] $PauseDuration = 1
+    )
+
+    do {
+        Write-Host "Current attempt: [$RetryAttempts]"
+        $result = Get-CommandResult -Command $Command -Multiline
+        Write-Host "Exit code: [$($result.ExitCode)]"
+        Write-Host "Output message: "
+        $($result.Output)
+        $RetryAttempts--
+        Start-Sleep -Seconds $PauseDuration
```

Comment:
Do we really need to sleep if exit code == 0 and command has finished successfully?

Reply:
Done